src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -101,40 +101,14 @@
     state: directory
     owner: "{{ cchq_user }}"
     group: "{{ cchq_user }}"
-    mode: "0755"
+    mode: 0755
   tags:
     - localsettings
     - formplayer_deploy
   with_items:
     - "{{ formplayer_build_dir }}"
     - "{{ [] if _should_update_formplayer_in_place else formplayer_release_dir }}"
-
-- name: Check if formplayer data dir exists
-  stat:
-    path: "{{ formplayer_data_dir }}"
-  tags:
-    - localsettings
-    - formplayer_deploy
-  register: check_data_dir
-
-- name: Formplayer data dir
-  file:
-    path: "{{ item }}"
-    state: directory
-    owner: "{{ cchq_user }}"
-    group: "{{ cchq_user }}"
-    mode: "0755"
-  tags:
-    - localsettings
-    - formplayer_deploy
-  with_items:
     - "{{ formplayer_data_dir }}"
-  when: not check_data_dir
-
-- name: Check if formplayer log dir exists
-  stat:
-    path: "{{ formplayer_access_log_dir }}"
-  register: check_access_log_dir
```

Comment:
oh, that puts both build and data directories in one task "Formplayer build dir". Apart from the confusion due to naming, I would still vote for having these different for better readability.
I think from the original code, task `Check if formplayer data dir exists` was not really needed.
Though I will defer to others better with CommCare Cloud code to review.

Reply:
Fixed that